From a05f428d987fadad1c2a1602d75245b42d6df89f Mon Sep 17 00:00:00 2001
From: Vagrant Cascadian <vagrant@debian.org>
Date: Tue, 14 May 2019 12:25:54 -0700
Subject: [PATCH] build/util/gen-hd-image: Start partition offset at 4MB to
 allow more room for boot firmware.

Boot firmware such as u-boot am335x_evm are installed at an offset
before the first partition, but has grown large enough to overlap and
thus overwrite the default 1MB partition offset.
---
 build/util/gen-hd-image | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build/util/gen-hd-image b/build/util/gen-hd-image
index 52697bec7..4a70ee2bd 100755
--- a/build/util/gen-hd-image
+++ b/build/util/gen-hd-image
@@ -63,7 +63,7 @@ PARTID="0x0c"
 FATSIZE="32"
 BUILDTYPE="complete"
 SOURCEDIR="."
-PARTOFFSET="2048"
+PARTOFFSET="8192"
 DEFAULT_IMAGESIZE="976560" # default d-i FLOPPY_SIZE for hd-media images
 IMAGESIZE="${DEFAULT_IMAGESIZE}"
 COMPRESS="none"
-- 
2.20.1

